libraries/AP_ESC_Telem/AP_ESC_Telem.cpp

```diff
@@ -502,12 +502,12 @@ void AP_ESC_Telem::update()
 
                 float rpm = 0.0f;
```

Comment:
At the risk of scope creeping you a second time now there floats we can use NaN. This means we can tell the difference between a genuine 0 rpm and unknown.

`float rpm = AP::logger().quiet_nanf()`

Reply:
Around this place, there is also a problem of using `uint32_t` for microseconds, so that we erroneously mark one RPM as stale roughly every 35 minutes... so yet another tiny fix.

(I am currently working on Extended DShot Telemetry v2 which brings various status bits and stress levels from ESC, which will also introduce new stuff in this location, so I will be there for a while anyway...)

Reply:
Just made a PR including this (#26252).